Source/WebCore/platform/graphics/texmap/TextureMapperPlatformLayer.h

```diff
@@ -46,6 +46,8 @@ class TextureMapperPlatformLayer {
     void setClient(TextureMapperPlatformLayer::Client* client) { m_client = client; }
 
     virtual bool isHolePunchBuffer() const { return false; }
+    virtual void notifyVideoPosition(const FloatRect&, const TransformationMatrix&) { };
+    virtual void paintTransparentRectangle(TextureMapper&, const FloatRect&, const TransformationMatrix&) { };
```

Comment:
Instead of adding these to the base class, they could be public in the HolePunch buffer and the caller just needs to cast before calling them.

Reply:
I thought this base class was CoordinatedPlatformLayerBuffer, but since it's TextureMapperPlatformLayer, the downcast is not possible, so let's leave it this way for now